### PR TITLE
i18n: localize 'Active:' text in timeline

### PR DESCRIPTION
**Pull request by MaikoTan**
@Akurosia @Jaehyuk-Lee May you have time to check if this localization correct?

### CODE REVIEW

**Comment by Jaehyuk-Lee** [comment]
Actually, I think Korean text to be remain as before, "Active: "
Some skill names are similar with "□□□: □□□□" but there's never "(English): (한국어)", always "(한국어): (한국어)".
English letters and Korean letters are very different so people's eye catches "Active: 스킬" only "스킬" part.
I don't think people be confused if "진행중 :" as a part of skill name though, people need to read and recognize it as it's not a skill name.

**Comment by MaikoTan** [comment]
Hmmmm...
Maybe we can add brackets on it, like `「진행중」: 긴 시전바`?
If I add `「进行中」` and `「進行」` on Chinese and Japanese timeline, folks would never confused because there are no `「」` in any skills.

**Comment by Jaehyuk-Lee** [comment]
Korean never use 「」 brackets except proper noun, for example, book name, thesis or paper. That's awkward in Korean, the `:` is correct with the "Active".
I'm asking for opinion on Korean community, I got new translation, `시전중`.

**Comment by Jaehyuk-Lee** [comment]
I don't play the game lately, I would myself run some raid with those changes if I have my account active.
There's no strong opinion on one side but slightly on the "let's translate" side.
So, let's first apply `시전중: (skill name)` this time, this is not a critical thing, let's see.

**Comment by MaikoTan** [comment]
I am now on the train, without a laptop. maybe you can comment a suggestion so that I can apply your request in one-click.

File: ui/raidboss/timeline.js [hunk 1]
```diff
@@ -53,6 +53,15 @@ const timelineInstructions = {
   ],
 };
 
+const activeText = {
+  en: 'Active:',
+  de: 'aktiv:',
+  fr: 'Active :',
+  ja: '(進行):',
+  cn: '(进行中):',
+  ko: 'Active:',
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/timeline.js:62]:
```suggestion
  ko: '시전중:',
```

Comment (Jaehyuk-Lee) [reply ui/raidboss/timeline.js:62]: I didn't know that suggesting a commit was this easy, I should use this 😃

File: ui/raidboss/timeline.js [hunk 2]
```diff
@@ -53,6 +53,15 @@ const timelineInstructions = {
   ],
 };
 
+const activeText = {
+  en: 'Active:',
+  de: 'aktiv:',
```
Comment (Akurosia) [inline ui/raidboss/timeline.js:58]: as it stand alone and just show a state, i would capitalize it to `Aktiv:`

**Pull request merged**